app/src/molecules/DeckInfoLabelTextTag/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { css } from 'styled-components'
+
+import {
+  ALIGN_CENTER,
+  BORDERS,
+  DISPLAY_GRID,
+  Flex,
+  FLEX_MAX_CONTENT,
+  JUSTIFY_END,
+  JUSTIFY_FLEX_START,
+  RESPONSIVENESS,
+  SPACING,
+  StyledText,
+} from '@opentrons/components'
+
+import type { ReactNode } from 'react'
+
+const MAX_SUPPORTED_LABELS = 3
+
+export interface DeckInfoLabelTextTagProps {
+  /* If more than MAX_SUPPORTED_LABELS labels are provided, only the first MAX_SUPPORTED_LABELS are rendered. */
+  colOneDeckInfoLabels: JSX.Element[]
+  colTwoText: ReactNode
+  /* Should always be a Tag. */
+  colThreeTag: JSX.Element
+}
+
+export function DeckInfoLabelTextTag({
+  colOneDeckInfoLabels,
+  colTwoText,
+  colThreeTag,
+}: DeckInfoLabelTextTagProps): JSX.Element {
+  return (
+    <Flex css={CONTAINER_STYLE}>
+      <Flex css={LABEL_CONTAINER_STYLE}>
+        {colOneDeckInfoLabels
+          .slice(0, MAX_SUPPORTED_LABELS)
+          .map(label => label)}
+      </Flex>
+      <StyledText
+        css={TEXT_STLYE}
+        oddStyle="bodyTextRegular"
+        desktopStyle="bodyDefaultRegular"
+      >
+        {colTwoText}
+      </StyledText>
+      <Flex css={TAG_CONTAINER_STYLE}>{colThreeTag}</Flex>
+    </Flex>
+  )
+}
+
+const CONTAINER_STYLE = css`
+  border-radius: ${BORDERS.borderRadius12};
+  padding: ${SPACING.spacing12};
+  width: 100%;
+  min-width: 26.875rem;
+  display: ${DISPLAY_GRID};
+  grid-template-columns: 120px 1fr ${FLEX_MAX_CONTENT};
+  gap: ${SPACING.spacing40};
+  align-items: ${ALIGN_CENTER};
+
+  @media ${RESPONSIVENESS.touchscreenMediaQuerySpecs} {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  @media (max-width: 450px) {
+    grid-template-columns: 1fr;
+    gap: ${SPACING.spacing12};
```

Comment:
From the design
![Screenshot 2025-02-28 at 2 59 21 PM](https://github.com/user-attachments/assets/ab4cdd10-0051-4dc5-83a4-8bfd99ae28a9)

```suggestion
    gap: ${SPACING.spacing8};
```

Reply:
Sorry, I forgot to link the correct breakpoints file:

https://www.figma.com/design/OIdG64Q5cgvEw82ish5Eee/Design-System%3A-Flex?node-id=12228-5428&t=6HLjhRbN2khVacds-4

I believe this one is the source of truth for the breakpoints.